### ISSUE TITLE
[Fabric 1.16.3] Invalid configuration generated

### ISSUE THREAD

**Issue by gdude2002** [TheRandomLabs/VanillaDeathChest#44]
**VDC version:** 1.11.0.7

On first start (or restart), an invalid configuration is generated. Note the second line here:

```toml
# VanillaDeathChest configuration
spawning.
[spawning]
	# Whether death chests should only be spawned if the container can be found in the player's inventory.
	# If this is enabled, the container is consumed if it is found.
	# Default: false
	useContainerInInventory = false
```

This, predictably, results in the following in the logs:

```
[10:47:28] [main/INFO]: [STDERR]: com.electronwill.nightconfig.core.io.ParsingException: Invalid bare key: 
[spawning
[10:47:28] [main/INFO]: [STDERR]: 	at com.electronwill.nightconfig.toml.TableParser.parseKey(TableParser.java:175)
[10:47:28] [main/INFO]: [STDERR]: 	at com.electronwill.nightconfig.toml.TableParser.parseDottedKey(TableParser.java:145)
[10:47:28] [main/INFO]: [STDERR]: 	at com.electronwill.nightconfig.toml.TableParser.parseNormal(TableParser.java:55)
[10:47:28] [main/INFO]: [STDERR]: 	at com.electronwill.nightconfig.toml.TomlParser.parse(TomlParser.java:44)
[10:47:28] [main/INFO]: [STDERR]: 	at com.electronwill.nightconfig.toml.TomlParser.parse(TomlParser.java:37)
```

If I comment out that second line, it's automatically uncommented after the server has started.

**Comment by TheRandomLabs** [comment]
Thanks for reporting! I'll check this out soon.

**Comment by TheRandomLabs** [comment]
Exams got in the way, but this should be fixed in the rewrite I have planned. :smile:

**Comment by TheRandomLabs** [comment]
Fixed in 2.0.0. I promise it'll be ready soon! 😛
